frontend/integration-tests/protractor.conf.ts

```diff
@@ -233,6 +233,7 @@ export const checkErrors = async () =>
   });
 
 export const firstElementByTestID = (id: string) => $(`[data-test-id=${id}]`);
+export const firstDropdownMenuByTestID = (id: string) => $(`[data-test-dropdown-menu=${id}]`);
```

Comment:
We probably want to move this into a file like `form.view.ts` instead of `protractor.conf.ts`

```suggestion
export const dropdownMenuForID = (id: string) => $(`[data-test-dropdown-menu=${id}]`);
```

Reply:
fixed